app/controllers/api/v0/forem_directories_controller.rb

```diff
@@ -0,0 +1,27 @@
+module Api
+  module V0
+    class ForemDirectoriesController < ApiController
+      def index
+        render json: {
+          cover_image_url: Settings::General.main_social_image,
+          description: Settings::Community.community_description,
+          logo_image_url: Settings::General.logo_png,
+          name: Settings::Community.community_name,
+          tagline: Settings::Community.tagline,
+          version: "edge.#{Time.current.strftime('%Y%m%d')}.0",
+          visibility: visibility
+        }.to_json,
+               status: :ok
+      end
+
+      private
+
+      def visibility
+        # TODO(ecosystem team) - add logic to determine if a Forem instance is suspended
```

Comment:
Do we have a way to "suspend" an entire Forem instance, yet? If so, we can add it now.

Reply:
Do you mean adding a way for the admins to opt-out from being listed despite the instance is public and running? If that's the case I think it makes sense. I don't think we have anything in place for that yet.

Returning "private" despite being public (based on a boolean setting) would mean it gets automatically unlisted, the moment the API endpoint is fetched. Just an idea that came to mind.